src/Components/Facility/MinQuantityList.tsx

```diff
@@ -75,8 +80,10 @@ export default function MinQuantityList(props: any) {
               {inventoryItem.item_object?.name}
             </p>
           </div>
-          <Link
-            href={`/facility/${facilityId}/inventory/${inventoryItem.id}/update/${inventoryItem.item_object?.id}`}
+          <Button
+            onClick={() => {
+              alert(3);
+            }}
```

Comment:
remove this if not needed

Reply:
Unintentional, was just used for debugging. Fixed now